sdcm/cluster_aws.py

```diff
@@ -322,6 +322,20 @@ def _wait_public_ip(self):
             self._instance.reload()
 
     def restart(self):
+        # We differenciate between "Restart" and "Reboot".
+        # Restart in AWS will be a Stop and Start of an instance.
+        # When using storage optimized instances like i2 or i3, the data on disk is deleted upon STOP. therefore, we
+        # need to setup the instance and treat it as a new instance.
+        if any(ss in self._instance.instance_type for ss in ['i3', 'i2']):
+            clean_script = dedent("""
```

Comment:
error': "global name 'dedent' is not defined"

--- 

fixed by PR https://github.com/scylladb/scylla-cluster-tests/pull/763